odl/discr/lp_discr.py

```diff
@@ -775,19 +778,43 @@ def show(self, title=None, method='', coords=None, indices=None,
             indices = []
             for axis, (xmin, xmax, n, coord) in enumerate(
                     zip(interv.min_pt, interv.max_pt, shape, coords)):
-                if coord is None:
-                    indices += [slice(None)]
+                try:
+                    coord_minp, coord_maxp = coord
+                except TypeError:
+                    coord_minp = coord_maxp = coord
+
+                # Validate input
```

Comment:
Cast `coord_minp` and `coord_maxp` already here to float. Otherwise, the comparison may raise and the error is less conclusive.